content/docs/1.6.0/references/storage-class-parameters.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Storage Class Parameters
+weight: 1
+---
+
+## Overview
+
+Storage Class as a resource object has a number of settable parameters.  Here's a sample YAML:
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: longhorn-test
+provisioner: driver.longhorn.io
+allowVolumeExpansion: true
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+parameters:
+  numberOfReplicas: "3"
+  staleReplicaTimeout: "2880"
+  fromBackup: ""
+  fsType: "ext4"
+  diskSelector: "ssd,fast"
+  nodeSelector: "storage,fast"
+#  recurringJobSelector: '[{"name":"snap-group", "isGroup":true},
+#                          {"name":"backup", "isGroup":false}]'
```

Comment:
Add all parameters here, but  comment out the optional parameters as below. However, you can give the optional parameter with the default value. 

```yaml
    parameters:
      numberOfReplicas: "{{ .Values.persistence.defaultClassReplicaCount }}"
      staleReplicaTimeout: "30"
      fromBackup: ""
      # fsType: ".."
      # mkfsParams: ".."
      # migratable: ".."
      ...
```